iam.tf

```diff
@@ -171,14 +171,14 @@ data "aws_iam_policy_document" "ecs" {
     sid       = "ECSAccess"
     effect    = "Allow"
     actions   = ["ecs:RunTask"]
-    resources = var.ecs_target_arns
+    resources = [replace(join(", ", var.ecs_target_arns), "/:\\d+$/", ":*")]
```

Comment:
Is it correct that `resources` will contain one element like `["arn...:*, arn:...*, arn:...*"]` ?

Please use 0.12+ syntax, like:
```hcl
resources = [for arn in var.ecs_target_arns: replace(arn, "/:\\d+$/", ":*")]
```

Reply:
done